rules/correct-component-name.js

```diff
@@ -0,0 +1,109 @@
+const { dim, bold } = require('colors');
+const { Rule } = require('../api');
+
+module.exports = class extends Rule {
+    getName() {
+        return 'correct-component-name';
+    }
+
+    checkTwigFile(component) {
+        const { twig } = component.files;
+        const {name: twigFileName} = twig;
+        const configDefinition = twig.api.external.definitions.filter(definition => definition.name === 'config')[0];
+        const { type, name, path } = component;
+        let contract;
```

Comment:
Why do you need this variable in this scope?

Reply:
It is my bad.
I declared this variable in another scope